core/src/main/scala/org/locationtech/rasterframes/expressions/package.scala

```diff
@@ -109,6 +110,7 @@ package object expressions {
     registry.registerExpression[ExpM1]("rf_expm1")
     registry.registerExpression[Sqrt]("rf_sqrt")
     registry.registerExpression[Resample]("rf_resample")
```

Comment:
this is actually a breaking change. SQL code that passed only two arguments before will break.

Reply:
Thanks for documenting it! I think it's fine.